config/application.rb

```diff
@@ -11,6 +11,7 @@ class Application < Rails::Application
   	include Cul::Omniauth::FileConfigurable
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
+    config.autoload_paths += %W(#{config.root}/lib)
```

Comment:
Looks like it isn't anymore http://guides.rubyonrails.org/autoloading_and_reloading_constants.html